### PR TITLE
Created Gyro Example Bot

### PR DESCRIPTION
**Pull request by Twinters007**

### CODE REVIEW

File: examples/examples/Gyro/robot.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from wpilib.joystick import Joystick
+
+class MyRobot(wpilib.SampleRobot):
+    
+    def robotInit(self):
+        gyroChannel = 0 #analog input
+        
+        self.joystickChannel = 0; #usb number in DriverStation
+    
+        #hannels for motors
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: hannels?

File: examples/examples/Gyro/robot.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        self.leftMotorChannel = 1
+        self.rightMotorChannel = 0
+        self.leftRearMotorChannel = 3
+        self.rightRearMotorChannel = 2
+    
+        self.angleSetpoint = 0.0
+        self.pGain = 1; #propotional turning constant
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: semicolon

File: examples/examples/Gyro/robot.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
+import wpilib
+from wpilib.joystick import Joystick
+
+class MyRobot(wpilib.SampleRobot):
+    
+    def robotInit(self):
+        gyroChannel = 0 #analog input
+        
+        self.joystickChannel = 0; #usb number in DriverStation
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: semicolon

File: examples/examples/Gyro/robot.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import wpilib
+from wpilib.joystick import Joystick
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: Why is this here? You don't need it. Just use wpilib.

File: examples/examples/Gyro/robot.py [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import wpilib
+
+class MyRobot(wpilib.SampleRobot):
+    
+    def robotInit(self):
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: Docstring what this does.

File: examples/examples/Gyro/robot.py [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import wpilib
+
+class MyRobot(wpilib.SampleRobot):
```
Comment (virtuald) [inline examples/examples/Gyro/robot.py]: Docstring what this class does.

**Comment by virtuald** [comment]
Add the two docstrings, and then rebase/squash it together, and we're golden. Thanks.

**Comment by Twinters007** [comment]
rebase?

**Comment by virtuald** [comment]
Yes, if you could turn this into a single commit, that would be great. Something like...

```
git rebase @~3
# in the commit editor, change each 'pick' to 's'
git push -f
```

See http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by Twinters007** [comment]
It wouldn't let me rebase. My index contained uncommitted changes, then when I tried to rebase afterward, it said I was up to date with master. Is this because my pull repo is robotpy-wpilib but my push is my fork?

**Comment by virtuald** [comment]
Oh. `git rebase -i @~3`
